From 06205619a01dbae9766967961ce7eaf9898c2042 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Thu, 29 Apr 2021 15:16:21 +0000
Subject: [PATCH] external: pass CMAKE_TOOLCHAIN_FILE

* fixes:
-- Building with code generator tool
CMake Error at /OE/build/luneos-honister/webos-ports/tmp-glibc/work/core2-64-webos-linux/anbox/3.0+gitAUTOINC+9de4e87cdd-r0/recipe-sysroot-native/usr/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:218 (message):
  Could NOT find EXPAT (missing: EXPAT_LIBRARY) (found version "2.2.10")
Call Stack (most recent call first):
  /OE/build/luneos-honister/webos-ports/tmp-glibc/work/core2-64-webos-linux/anbox/3.0+gitAUTOINC+9de4e87cdd-r0/recipe-sysroot-native/usr/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:582 (_FPHSA_FAILURE_MESSAGE)
  /OE/build/luneos-honister/webos-ports/tmp-glibc/work/core2-64-webos-linux/anbox/3.0+gitAUTOINC+9de4e87cdd-r0/recipe-sysroot-native/usr/share/cmake-3.19/Modules/FindEXPAT.cmake:63 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  tools/CMakeLists.txt:15 (find_package)

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Upstream-Status: Pending
---
 external/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/external/CMakeLists.txt b/external/CMakeLists.txt
index d50f1a6..163d21b 100644
--- a/external/CMakeLists.txt
+++ b/external/CMakeLists.txt
@@ -14,4 +14,5 @@ ExternalProject_Add(sdbus-cpp
   CMAKE_CACHE_ARGS
         -DBUILD_CODE_GEN:BOOL=ON
         -DBUILD_SHARED_LIBS:BOOL=OFF
+  CMAKE_ARGS -DCMAKE_TOOLCHAIN_FILE=${PROJECT_SOURCE_DIR}/../toolchain.cmake
 )
